bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/util/BLangConstants.java

```diff
@@ -99,7 +99,7 @@
 
     public static final Module BALLERINA_BUILTIN_PKG_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX, "builtin");
     public static final Module BALLERINA_RUNTIME_PKG_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX,
-                                                                     "runtime", "0.5.0");
+                                                                     "runtime", "0.5.1");
```

Comment:
What is the usage of this constant? . can't we remove this

Reply:
Now, we can't remove this as [getCallStack()](https://github.com/ballerina-platform/module-ballerina-runtime/blob/master/runtime-native/src/main/java/org/ballerinalang/stdlib/runtime/nativeimpl/GetCallStack.java#L35) is using this. So we need to implement this method properly.